typescript/infra/config/environments/testnet3/agent.ts

```diff
@@ -117,7 +117,7 @@ const releaseCandidate: RootAgentConfig = {
   context: Contexts.ReleaseCandidate,
   rolesWithKeys: [Role.Relayer, Role.Kathy, Role.Validator],
   relayer: {
-    connectionType: AgentConnectionType.HttpFallback,
+    rpcConsensusType: RpcConsensusType.Fallback,
```

Comment:
How about `RpcPolicy` instead of `RpcConsensusType`? Or maybe `RpcConnectionType` to match the helm chart

Reply:
I would rather get away from `rpcConnectionType` since it implies the old choice of `httpFallback, httpQuorum, http, ..`, and I went with `rpcConsensusType` at @tkporter's suggestion.